src/scripts/components/main/app/app.tsx

```diff
@@ -36,12 +37,21 @@ const App: FunctionComponent = () => (
 );
 
 const TranslatedApp: FunctionComponent = () => {
-  const language = useSelector(languageSelector);
   const markers = useStoryMarkers();
+  const language = useSelector(languageSelector);
+
+  const getLanguage = () => {
+    if (localStorage.getItem('language')) {
+      return localStorage.getItem('language') as Language;
```

Comment:
We already retrieve the browser language in the language reducer. This would be a good place to also check for the localStorage language. It also has the benefit that it is only evaluated once on app start and not on every render like here. 

Also we have to check if the language in the storage is actually one of the available languages similar to the getBrowserLanguage function.